_data/locales/id.yml

```diff
@@ -0,0 +1,31 @@
+id:
+  locale_name: Indonesian
```

Comment:
@willysr Actually: just noticed this. Could you update this so it's the Indonesian name for Indonesian? Thanks!

Reply:
sure, i will send another PR in a minute

thanks

Reply:
sent as https://github.com/github/opensource.guide/pull/547